content/examples/shortcodes/code-tabs/en.md

```diff
@@ -68,7 +68,7 @@ languages:
   name: Norwegian
 {{< /code-tab >}}
 
-{{< code-tab name="data.yaml" language="yaml" linenos="true" area="bottom-left" >}}
+{{< code-tab name="data.yaml" language="yaml" linenos="true" area="bottom-left" fileIcon="false" >}}
```

Comment:
also update the code-isage example above with the same so the example and result are the same

Reply:
Check!